modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -1,48 +1,27 @@
 package {{packageName}}.infrastructure
 
-import okhttp3.Credentials
-import okhttp3.OkHttpClient
-import okhttp3.RequestBody
-{{#jvm-okhttp3}}
-import okhttp3.MediaType
-{{/jvm-okhttp3}}
-{{#jvm-okhttp4}}
-import okhttp3.RequestBody.Companion.asRequestBody
-import okhttp3.RequestBody.Companion.toRequestBody
-{{/jvm-okhttp4}}
-import okhttp3.FormBody
-{{#jvm-okhttp3}}
-import okhttp3.HttpUrl
-{{/jvm-okhttp3}}
+import okhttp3.*
```

Comment:
I'm not the biggest fan o star imports, but maybe this is only a matter of tast

Reply:
@saschpe although this may be a controversial subject, after reading a bit more on the subject, I think I would prefer keep the previous structure.
The main reason for this is because when reviewing PR's here on github, and we don't have the IDE to help figure out where the types comes from, so I would prefer to continue doing the imports explicitly.
So could you please revert the imports, to avoid using star import's?
Thanks

Reply:
I don't mind much either way but this was an IntelliJ recommendation. Type conflicts on imports usually lead to compile errors / symbol mismatches. Upstream Ktor also uses star imports in their documentation and samples.

Reply:
Yes, I know, and kotlin client multiplatform also uses it.
But since kotlin cliente jvm doesn't use them I would prefer to keep it that way.
Although this is something that may eventually change in future, because there is no obvious right or wrong answer here.

Reply:
IntelliJ disagrees, with the current code, it displays multiple unused import warnings in various files...